### PR TITLE
[15.0][IMP] sale_elaboration: Allow to select multiple elaborations

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa TT41408

Extends the functionality of the sale_elaboration module to be able to specify in a sales order line not only one elaboration, but several (migrations scripts included).

### CODE REVIEW

**Comment by ernestotejeda** [comment]
ping @sergio-teruel @pedrobaeza @CarlosRoca13

**Comment by ernestotejeda** [comment]
Ready to review

File: product_supplierinfo_for_customer_elaboration/tests/test_customerinfo_elaboration.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def setUpClass(cls):
             {
                 "name": cls.customer0.id,
                 "product_tmpl_id": cls.product.product_tmpl_id.id,
-                "elaboration_id": cls.elaboration.id,
+                "elaboration_ids": cls.elaboration,
```
Comment (pedrobaeza) [inline product_supplierinfo_for_customer_elaboration/tests/test_customerinfo_elaboration.py]: I think this is working by chance.

```suggestion
                "elaboration_ids": [(4, cls.elaboration.id)],
```

File: sale_elaboration/__manifest__.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 {
     "name": "Sale Elaboration",
     "summary": "Set an elaboration for any sale line",
-    "version": "15.0.1.0.1",
+    "version": "15.0.2.0.1",
```
Comment (pedrobaeza) [inline sale_elaboration/__manifest__.py]: It should be (and so the migration folder):

```suggestion
    "version": "15.0.2.0.0",
```

**Review by pedrobaeza — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]
Functional review OK
Great job!! @ernestotejeda

**Review by CarlosRoca13 — APPROVED** [review]

**Comment by ernestotejeda** [comment]
@pedrobaeza changes donde

**Comment by pedrobaeza** [comment]
I'm afraid you have amended the changes on the incorrect commit, @ernestotejeda

**Comment by ernestotejeda** [comment]
@pedrobaeza fixed 
:facepalm:

**Pull request merged**